outputs/gcp_cloudaccount/output.facets.yaml

```
@@ -0,0 +1,17 @@
+name: gcp_cloud_account
+out:
+  type: object
+  title: Cloud Account
+  description: Cloud Account
+  properties:
+    attributes:
+    interfaces:
+      type: object
+      description: Attributes of the gcp cloud account module outputs
+      properties:
+          project:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)